edx/completion
# Add code to allow xblocks to submit completions for arbitrary users.

**Description:** Describe in a couple of sentence what this PR adds

**JIRA:** MCKIN-8253 BB-132

**Dependencies:** N/A

**Installation instructions:** 

**Testing instructions:** 

Modify an xblock to call the submit_completion code in the completion service.  Verify that the completions get created / updated appropriately

**Reviewers:**
- [ ] @__xitij2000__

**Merge checklist:**

**Post merge:**
      finished.
